core/src/main/java/io/spine/core/MessageWithContext.java

```diff
@@ -47,10 +54,31 @@
      */
     MessageContext getContext();
 
+    /**
+     * Obtains the unpacked form of the enclosed message.
+     *
+     * @see #getMessage()
+     */
+    default Message enclosedMessage() {
```

Comment:
Please take a look at the `Events` and `Commands` utilities. Those define methods `getMessage()` for the same thing. I suggest we deprecate those methods (or maybe, the entire utility classes).

Reply:
We cannot get rid of these utilities just yet. The `enclosedMessage()` returns `Message`, and utilities return specific types like `CommandMessage` and `EventMessage`. The Model Compiler does not support passing generic parameters, so I don't know how we'd overload the `enclosedMessage()` method returning more narrow type from `Command` or `Event`.